Formula/i/intercal.rb

```diff
@@ -1,8 +1,8 @@
 class Intercal < Formula
   desc "Esoteric, parody programming language"
   homepage "http://catb.org/~esr/intercal/"
-  url "http://catb.org/~esr/intercal/intercal-0.31.tar.gz"
-  sha256 "93d842b81ecdc82b352beb463fbf688749b0c04445388a999667e1958bba4ffc"
+  url "http://catb.org/~esr/intercal/intercal-0.32.tar.gz"
```

Comment:
**suggestion:** Could replace the non-working homepage with its source repo: https://gitlab.com/esr/intercal, perhaps also replace the `url`, too.

Reply:
yeah, I think we can still use the catb.org url, there are many other software on the same site.